code/modules/uplink/uplink_item.dm

```diff
@@ -1017,6 +1017,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 /datum/uplink_item/cyber_implants
 	category = "Cybernetic Implants"
 	surplus = 0
+	refundable = TRUE
```

Comment:
No, this is awful anyways. The only refundable items atm are ones that require player ghosts to be available to even function. Please give a reason as to why implants should be refundable.